ActindoWorkFlow/Operation/01_ManageWorkflows.md

```diff
@@ -0,0 +1,423 @@
+
+3. Click the ![Points](/Assets/Icons/Points02.png "[Points]") (Points) button in the upper left corner next to the workflow name.   
+  The workflow context menu is displayed.
+
+  ![Context menu](/Assets/Screenshots/ActindoWorkFlow/Workflows/ContextMenu.png "[Context menu]")
+
+4. Click the *Import JSON ..* menu entry in the context menu.   
+  The *Import JSON* window with the JSON code for the current workflow is displayed.
+
+  ![Import JSON](/Assets/Screenshots/ActindoWorkFlow/Workflows/ImportJSON.png "[Import JSON]")
+
+5. Select the complete JSON code in the window, replace it by the JSON code in your clipboard and click the [IMPORT JSON] button in the bottom right corner.
+  The *Import JSON* window is closed. The copied workflow is displayed in the workflow editor.
+
+  ![Workflow editor copied](/Assets/Screenshots/ActindoWorkFlow/Workflows/WorkflowEditorCopied.png "[Workflow editor copied]")
+
+  > [Info] You can edit the workflow as desired before deploying it.
+
+6. Click the ![Points](/Assets/Icons/Points02.png "[Points]") (Points) button in the upper left corner next to the workflow name.   
+  The workflow context menu is displayed.
+
+  ![Context menu](/Assets/Screenshots/ActindoWorkFlow/Workflows/ContextMenu.png "[Context menu]")
+
+7. Click the *Deploy* menu entry in the context menu.   
+  The copied workflow is deployed and published.
+
+### Next steps
+
+- [Publish a workflow version](#publish-a-workflow-version)
+- [Unpublish a workflow version](#unpublish-a-workflow-version)
+- [Track a workflow process](02_TrackProcesses.md)
+
+### See also
+
+- [User Interface workflows](/ActindoWorkFlow/UserInterface/00_UserInterface.md)
+- [Create a workflow](#create-a-workflow)
+- [Edit a workflow](#edit-a-workflow)
+- [Workflow and process elements](/ActindoWorkFlow/Overview/02_WorkflowProcessElements.md)
+
+
+
+## Publish a workflow version
+
+A workflow can only be used for processes if it is published. If multiple versions of a workflow exist, the highest published version is taken for upcoming workflow processes. Therefore you have to publish a workflow to use it for workflow processes. By default, a created workflow or a deployed workflow version is automatically published. You can manually publish or [unpublish](#unpublish-a-workflow-version) workflow versions.
+
+### Prerequisites
+
+- At least one workflow is created, see [Create a workflow](#create-a-workflow).
+- At least one workflow version is unpublished, see [Unpublish a workflow version](#unpublish-a-workflow-version).
+
+### Procedure
+
+*Workflows > Workflows > Tab OVERVIEW > Select workflow*
+
+![Workflow versions published](/Assets/Screenshots/ActindoWorkFlow/Workflows/WorkflowVersionsPublished.png "[Workflow versions published]")
+
+1. Select the checkbox of the workflow version you want to publish.      
+  The editing toolbar is displayed.
+
+2. Click the [UNPUBLISH] button in the editing toolbar.   
```

Comment:
``Click the [UNPUBLISH] button in the editing toolbar. 

Sollte es nicht [PUBLISH] sein? Abschnitt heißt "Publish a workflow version". Ähnliches gilt für nächsten Abschnitt. Anderum oder habe ich falsch verstanden?